### PR TITLE
Feature/80 allow web frontend to load apps version from web backend

### LINKED ISSUE #80: Feature: Allow Web Frontend to load app's version from Web Backend
**Is your feature request related to a problem? Please describe.**
For development purpous, version value displayed on Web Frontend is mocked.

**Describe the solution you'd like**

- Create logic to get `Version` class's values on Web Backend, return `Major`, `Minor` and `Patch` properties,
- Create Service and Controller classes for this value,
- Replace mocked value with the one returned from Web Backend,

**Describe alternatives you've considered**

**Additional context**
Web Frontend
Web Backend

### PR DESCRIPTION
**Pull request by Lukash88**
## Summary

The app's version is now loaded dynamically from Web Backend.

## Details

- Make Service for retrieving version
- Replace mocked value with the one returned from Web Backend

**Screenshot:**

image

## References

No external references needed.

### CODE REVIEW

File: binder-web-frontend/src/pages/home/components/header/header.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <div class="header">
-    <p class="header__title">Binder <span class="header__title-version">v{{ version }}</span></p>
+    <p class="header__title">Binder <span class="header__title-version">v{{ version }}</span>
+    </p>
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.html]: Move this tag back to upper line.

File: binder-web-frontend/src/pages/home/components/header/header.component.ts [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
   selector: 'header',
   templateUrl: './header.component.html',
-  styleUrls: ['./header.component.scss']
+  styleUrls: ['./header.component.scss'],
 })
-export class HeaderComponent { 
-   version: number = 0;
+export class HeaderComponent implements OnInit, OnDestroy {
+  private subscribe$: Subject<void> = new Subject<void>();
+  attributes: string[] = ['major', 'minor', 'patch', 'name'];
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.ts]: Are this values used somewhere?

File: binder-web-frontend/src/pages/home/components/header/header.component.ts [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
+  //version?: DisplayableVersion;
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.ts]: Remove comment, if not used.

File: binder-web-frontend/src/pages/home/components/header/header.component.ts [hunk 4]
```diff
@@ -1,10 +1,36 @@
...
+  version?: string;
+
+  constructor(private appVersionsService: AppVersionsService) {}
+
+  ngOnInit(): void {
+    this.appVersionsService
+      .getVersionGet()
+      .pipe(take(1))
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.ts]: Connect subscription control here.

File: binder-web-frontend/src/pages/home/components/header/header.component.ts [hunk 5]
```diff
@@ -1,10 +1,36 @@
...
+      .subscribe({
+        next: (ver: string) => 
+
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.ts]: Clean up code.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-frontend/src/pages/home/components/header/header.component.html (hunk 1 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.html]: Resolved 6b25ad1

File: binder-web-frontend/src/pages/home/components/header/header.component.ts (hunk 3 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.ts]: Resolved 6b25ad1

File: binder-web-frontend/src/pages/home/components/header/header.component.ts (hunk 2 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.ts]: Resolved 6b25ad1

File: binder-web-frontend/src/pages/home/components/header/header.component.ts (hunk 5 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.ts]: Resolved 6b25ad1

File: binder-web-frontend/src/pages/home/components/header/header.component.ts (hunk 4 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.ts]: Resolved 6b25ad1

File: binder-web-backend/Binder.Persistence/Migrations/20231022154458_AddAppVersionEntity.Designer.cs
Comment (StraykerPL) [inline binder-web-backend/Binder.Persistence/Migrations/20231022154458_AddAppVersionEntity.Designer.cs:1]: Clean up migrations, this version value is not in DB anymore.

File: binder-web-frontend/src/pages/home/components/header/header.component.ts [hunk 6]
```diff
@@ -1,10 +1,32 @@
...
   selector: 'header',
   templateUrl: './header.component.html',
-  styleUrls: ['./header.component.scss']
+  styleUrls: ['./header.component.scss'],
 })
-export class HeaderComponent { 
-   version: number = 0;
+export class HeaderComponent implements OnInit, OnDestroy {
+  private subscribe$: Subject<void> = new Subject<void>();
+  version?: string;
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/header/header.component.ts]: Make this value non-nullable, add value on definition's level.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-backend/Binder.Persistence/Migrations/20231022154458_AddAppVersionEntity.Designer.cs
Comment (Lukash88) [reply binder-web-backend/Binder.Persistence/Migrations/20231022154458_AddAppVersionEntity.Designer.cs:1]: Resolved 1c21729

File: binder-web-frontend/src/pages/home/components/header/header.component.ts (hunk 6 above)
Comment (Lukash88) [reply binder-web-frontend/src/pages/home/components/header/header.component.ts]: Resolved 09b64a7

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**